cleanup_all_covid_calls.py

```diff
@@ -0,0 +1,155 @@
+import pandas as pd
+from uszipcode import SearchEngine
+import numpy as np
+from datetime import datetime
+
+file = "Data from 4.2.20 Fake Data.xlsx"
+
+# read all sheets, returns a dict of dataframes
+dfs = pd.read_excel(file, sheet_name=None)
+
+### Cleanup for All COVID Calls dashboard
+
+# step 1
+# select required columns from VIA LINK’s Disaster Form
+# pretty sure the distaster form is "Uncleaned data type 1 VIA LINK"
+VIA_LINK_REQUIRED_COLUMNS_DISASTER = [
+    "CallReportNum",
+    "ReportVersion",
+    "CallDateAndTimeStart",
+    "CityName",
+    "CountyName",
+    "StateProvince",
+    "PostalCode",
+    "Client Information - Age Group",
+    "Client Information - Call Type",
+    "Client Information - Identifies as",
+    "Concerns/Needs - Concerns/Needs",
+    "Contact Source - Program ",  # ending space is needed
+    "Needs - Basic Needs Requested",
+]
+vialink1_df = dfs["Uncleaned data type 1 VIA LINK"][VIA_LINK_REQUIRED_COLUMNS_DISASTER]
+
+# step 2
+# select required columns from 232-Help’s Disaster Form
+TWO32_HELP_REQUIRED_COLUMNS = [
+    "CallReportNum",
+    "ReportVersion",
+    "CallDateAndTimeStart",
+    "CityName",
+    "CountyName",
+    "StateProvince",
+    "PostalCode",
+    "Client Information - Date of Birth",
+    "Client Information - Call Type",
+    "Call Outcome - What concerns/needs were identified?",
+    "Client Information - Identifies as",
+    "Needs - Basic Needs Requested",
+]
+two32_help_df = dfs["Uncleaned Data from 232-Help"][TWO32_HELP_REQUIRED_COLUMNS]
+
+# step 3
+# Create age ranges from date of birth
+# use ranges 0-5, 6-12, 13-17, 18-24, 25-40, 41-59, 60+.
+now = datetime.now()
+bins = [0, 5, 12, 17, 24, 40, 59, 150]
+labels = ["0-5", "6-12", "13-17", "18-24", "24-40", "41-49", "60+"]
+dob = pd.to_datetime(
+    two32_help_df["Client Information - Date of Birth"], errors="coerce"
+)
+years_old = (now - dob).astype("timedelta64[Y]")
+age_range = pd.cut(years_old, bins=bins, labels=labels, include_lowest=True)
+two32_help_df["Client Information - Age Group"] = age_range
+# remove original Date of Birth column
+two32_help_df.drop(columns=["Client Information - Date of Birth"], inplace=True)
```

Comment:
Nit: I'm not sure how I feel about using `inplace=True` yet.  My FP preferences make me want to avoid any state mutation, but I'm starting to think it's probably ok for scripts and wrangling.  I do find it more readable sometimes so maybe it's worthwhile

Reply:
For `inplace=True`, sometimes it's required for memory purposes (so as not to make an entire new copy of the data in memory). It also avoids an allocation step, which means it runs a bit faster. For smaller datasets, it doesn't really matter that much.